src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -98,11 +132,39 @@ const SidebarView = View.extend({
     this.showGroups();
     this.showInfo();
   },
+  onClickMenu() {
+    const menuOptions = new Backbone.Collection([
+      {
+        onSelect: _.bind(this.triggerMethod, this, 'confirm:delete'),
+      },
+    ]);
+
+    const optionlist = new Optionlist({
+      ui: this.ui.menu,
+      uiView: this,
+      headingText: i18n.sidebarView.menuOptions.headingText,
+      itemTemplate: hbs`<span class="sidebar__delete-icon">{{far "trash-alt"}}</span>{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.sidebarView.menuOptions.delete }}`,
+      lists: [{ collection: menuOptions }],
+      align: 'right',
+      popWidth: 248,
+    });
+
+    optionlist.show();
+  },
+  canEdit() {
+    return this.model.isNew() || !this.model.get('last_active_at');
+  },
   showName() {
-    this.showChildView('name', new NameView({ model: this.model }));
+    this.showChildView('name', new NameView({
+      model: this.model,
+      canEdit: this.canEdit(),
```

Comment:
if we move `canEdit` to the model, we won't need to pass it here.  these views can just `this.model.canEdit`